src/controllers/locales/en.json

```diff
@@ -1,100 +1,115 @@
+	"Validation error: email is not allowed to be empty": "Validation error: email is not allowed to be empty",
+	"Validation error: nationalId must be a safe number": "Validation error: nationalId must be a safe number",
+	"Validation error: role is required": "Validation error: role is required",
+	"Validation error: email is not allowed": "Validation error: email is not allowed",
+	"Validation error: nationalId is not allowed": "Validation error: nationalId is not allowed",
+	"Validation error: id is not allowed": "Validation error: id is not allowed",
+	"Role operator is not allowed": "Role operator is not allowed",
+	"Role driver is not allowed": "Role driver is not allowed",
+	"Role admin is not allowed": "Role admin is not allowed",
+	"Validation error: role is not allowed": "Validation error: role is not allowed",
+	"Validation error: phone length must be at least 10 characters long": "Validation error: phone length must be at least 10 characters long",
+	"Please sign in as Admin": "Please sign in as Admin",
+	"Bus created successfully": "Bus created successfully",
+	"There was an error while creating a bus": "There was an error while creating a bus",
+	"There are no buses registered in the system": "There are no buses registered in the system",
+	"Bus not found": "Bus not found",
+	"Bus already exists in the system.": "Bus already exists in the system.",
+	"Bus found": "Bus found",
+	"Bus updated successfully": "Bus updated successfully",
+	"Bus not updated ": "Bus not updated ",
+	"Bus not registered in the system ": "Bus not registered in the system ",
+	"Bus deleted successfully": "Bus deleted successfully",
+	"Bus not found ": "Bus not found ",
+	"Bus created successfully.": "Bus created successfully.",
+	"Bus assigned to route successfully": "Bus assigned to route successfully",
+	"That bus is not registered": "That bus is not registered",
+	"That route is not found": "That route is not found",
+	"there was an error while assigning bus": "there was an error while assigning bus",
+	"This page does not exist": "This page does not exist",
+	"These are  buses assigned to routes": "These are  buses assigned to routes",
+	"there was an error while getting all assigned buses": "there was an error while getting all assigned buses",
+	"Bus is already assigned to this route": "Bus is already assigned to this route",
+	"There was an error while assigning bus": "There was an error while assigning bus",
+	"Bus is not yet assigned to any route": "Bus is not yet assigned to any route",
+	"unAssigned bus from route successfully": "unAssigned bus from route successfully",
+	"There was an error while unassigning bus": "There was an error while unassigning bus",
+	"Validation error: routeID is not allowed": "Validation error: routeID is not allowed",
+	"Validation error: distance must be a string": "Validation error: distance must be a string",
+	"Validation error: busStops is not allowed": "Validation error: busStops is not allowed"
+}
```

Comment:
let's have an empty line after this file

Reply:
Yeah sure let me work us that